### PR TITLE
Expanding on the SMES setup logic.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Some minor tweaks and an additional landmark ported from the Polaris engine setup logic.

### CODE REVIEW

File: code/modules/supermatter/setup_supermatter.dm [hunk 1]
```diff
@@ -206,13 +214,24 @@ var/global/list/engine_setup_markers = list()
...
 	S.output_attempt = 1
-	S.input_level = S.input_level_max
-	S.output_level = S.output_level_max
+	if(target_input_level)
+		if(target_input_level > S.input_level_max)
+			S.input_level = S.input_level_max
+		else
+			S.input_level = target_input_level
+	else
+		S.input_level = S.input_level_max
```
Comment (afterthought2) [inline code/modules/supermatter/setup_supermatter.dm]: How about `S.input_level = min(target_input_level, S.input_level_max)` and default `target_input_level` to infinity?

File: code/modules/supermatter/setup_supermatter.dm [hunk 2]
```diff
@@ -247,6 +266,27 @@ var/global/list/engine_setup_markers = list()
 	F.update_use_power(POWER_USE_IDLE)
 	return SETUP_OK
 
+// Closes the monitoring room shutters so the first Engi to show up doesn't get microwaved
```
Comment (afterthought2) [inline code/modules/supermatter/setup_supermatter.dm:256]: I don't really get it; can't you just map them closed?

Comment (MistakeNot4892) [reply code/modules/supermatter/setup_supermatter.dm:256]: I kind of mindlessly copied this over from Polaris when converting the map, this seems like a valid point.

Comment (MistakeNot4892) [reply code/modules/supermatter/setup_supermatter.dm:256]: Thinking about it - maybe you want closing the shutters to be part of the routine, I guess. I don't think it causes any harm to include this landmark.

**Review by afterthought2 — APPROVED** [review]

File: code/modules/supermatter/setup_supermatter.dm (hunk 2 above)
Comment (afterthought2) [reply code/modules/supermatter/setup_supermatter.dm:256]: I mean, it doesn't even close them, it toggles them. But OK.

**Pull request merged**